### PR TITLE
Add test for bootstrap.py

### PR DESCRIPTION
**Pull request by swadeley**
This  adds test for bootstrap.py, a.k.a the bootstrap script.

### CODE REVIEW

**Comment by swadeley** [comment]
robottelo]$ pytest --show-capture=no tests/foreman/cli/test_bootstrap_script.py
=========================== test session starts ==========================================
platform linux -- Python 3.8.7, pytest-6.2.1, py-1.9.0, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /sat6/robottelo_1/robottelo, configfile: pyproject.toml
plugins: xdist-2.2.0, mock-3.4.0, forked-1.3.0, services-2.2.1, ibutsu-1.13, cov-2.10.1
collected 1 item                                                                                                                                                                                  

tests/foreman/cli/test_bootstrap_script.py .                                                                                                                                                [100%]

================================== 1 passed, 19 warnings in 268.34s (0:04:28) =============

**Review by mshriver — APPROVED** [review]
Looks like a great test to me! Thanks @swadeley

File: tests/foreman/cli/test_bootstrap_script.py [hunk 1]
```diff
@@ -15,54 +15,73 @@
...
+        # assure system is not registered
+        result = vm.execute('subscription-manager identity')
+        # result will be 1 if not registered
+        assert result.status == 1
+        assert vm.execute(
+            f'curl -o /root/bootstrap.py "http://{settings.server.hostname}/pub/bootstrap.py" '
+        )
+        assert vm.execute(
+            f'python /root/bootstrap.py -s "{settings.server.hostname}" -o {module_org.name} '
+            f'-L Default_Location -a {activation_key.name} --hostgroup={hg.name}'
```
Comment (jameerpathan111) [inline tests/foreman/cli/test_bootstrap_script.py:71]: @swadeley should we use module_location here?

Comment (mshriver) [reply tests/foreman/cli/test_bootstrap_script.py:71]: @jameerpathan111 correct, the location here shouldn't be hard-coded, this should be a blocking comment.

Comment (swadeley) [reply tests/foreman/cli/test_bootstrap_script.py]: sorry about that, now fixed

**Comment by swadeley** [comment]
I moved the module activation key to the `pytest fixtures/apipytest fixtures/api` file and renamed it `module_ak` as suggested elsewhere by @rplevka

**Comment by swadeley** [comment]
robottelo]$ pytest  --show-capture=no tests/foreman/cli/test_bootstrap_script.py -k test_positive_register
============================= test session starts ===============================================
platform linux -- Python 3.8.7, pytest-6.2.1, py-1.9.0, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/sjw/sat6/robottelo_1/robottelo, configfile: pyproject.toml
plugins: xdist-2.2.0, forked-1.3.0, services-2.2.1, cov-2.11.1, ibutsu-1.13, mock-3.5.1
collected 1 item                                                                                                                                                                                  

tests/foreman/cli/test_bootstrap_script.py .                                                                                                                                                [100%]


================================ 1 passed, 18 warnings in 220.24s (0:03:40) =================

**Comment by omkarkhatavkar** [comment]
Can one of the admins verify this patch?

**Review by jameerpathan111 — APPROVED** [review]
ACK

**Comment by swadeley** [comment]
Thank you all

**Comment by swadeley** [comment]
Thank you

**Pull request merged**